### PR TITLE
feat: main function docstring and default arguments

### PR DESCRIPTION
**Pull request by leoporoli**

### CODE REVIEW

File: main.star [hunk 1]
```diff
@@ -4,9 +4,45 @@ express_backend = import_module("/backend/backend.star")
 frontend = import_module("/frontend/frontend.star")
 
 
-def run(plan, args):
+def run(plan, mongo_config = None, backend_service_config = None):
+    """Starts this MERN example application.
```
Comment (h4ck3rk3y) [inline main.star:8]: does this doc string pass our validation?

Comment (leoporoli) [reply main.star]: now is valid, thanks for asking

**Comment by leoporoli** [comment]
@h4ck3rk3y now is ready for the review

**Comment by leoporoli** [comment]
@h4ck3rk3y don't forget this guy

**Review by h4ck3rk3y — APPROVED** [review]
Looks good to me!

**Pull request merged**